src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -135,47 +148,57 @@ export const RegistrationFormActions = ({
           onClick={toggleSupportModal}>
           {t('my_page.messages.get_curator_support')}
         </Button>
-        {!isLastTab ? (
-          <Box
-            sx={{
-              gridArea: 'save-button',
-              display: 'flex',
-              justifyContent: 'end',
-              alignItems: 'center',
-            }}>
+
+        <Box
+          sx={{
+            gridArea: 'action-buttons',
+            display: 'flex',
+            justifyContent: 'end',
+            alignItems: 'center',
+          }}>
+          <Button
+            data-testid={dataTestId.registrationWizard.formActions.cancelEditButton}
+            onClick={cancelEdit}
+            color="primary"
+            sx={{ width: 'fit-content', justifySelf: 'center' }}>
+            {t('common.cancel')}
+          </Button>
+          {!isLastTab ? (
             <LoadingButton
               variant="outlined"
               loading={isSaving}
               data-testid={dataTestId.registrationWizard.formActions.saveRegistrationButton}
               onClick={handleSaveClick}>
               {t('common.save')}
             </LoadingButton>
-            <Tooltip title={t('common.next')} sx={{ gridArea: 'next-button' }}>
-              <IconButton
-                onClick={() => setTabNumber(tabNumber + 1)}
-                data-testid={dataTestId.registrationWizard.formActions.nextTabButton}>
-                <KeyboardArrowRightIcon
-                  sx={{
-                    color: 'white',
-                    borderRadius: '50%',
-                    bgcolor: 'primary.light',
-                    height: '1.875rem',
-                    width: '1.875rem',
-                  }}
-                />
-              </IconButton>
-            </Tooltip>
-          </Box>
-        ) : (
-          <LoadingButton
-            variant="contained"
-            loading={isSaving}
-            data-testid={dataTestId.registrationWizard.formActions.saveRegistrationButton}
-            onClick={handleSaveClick}
-            sx={{ gridArea: 'save-button', width: 'fit-content', justifySelf: 'end' }}>
-            {t('common.save_and_view')}
-          </LoadingButton>
-        )}
+          ) : (
+            <LoadingButton
+              variant="contained"
+              loading={isSaving}
+              data-testid={dataTestId.registrationWizard.formActions.saveRegistrationButton}
+              onClick={handleSaveClick}
+              sx={{ width: 'fit-content', justifySelf: 'end' }}>
```

Comment:
Samme som over. Mulig det var nødvendig før, men ikke nå? Eller overser jeg noe?
```suggestion
              >
```